spec/requests/membership_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+describe Membership do
+  it 'delegation became false when archive' do
+    user = create(:user, :who_delegated)
+    membership = user.primary_membership
+
+    expect(membership.user.delegated).to be true
```

Comment:
Usually every test has 4 phases:
- setup
- exercise
- verify
- teardown

In the current example, rspec and database cleaner take care about the last one. In my opinion this check doesn’t really have much added value before the exercise, we don’t want to verify the setup, but the exercise itself. It also adds some noise making harder to understand. I’d delete all the checks before the exercises and just keep the ones after it. What is your opinion?

Reply:
I agree. It's a bit like we test the fixture, not our logic, which isn't a good practice in my opinion. I tried to keep the tests as close to the originals as possible, because I wasn't too confident with RSpec.  I'll remove it.